pyrobosim_ros_gym/eval.py

```diff
@@ -9,20 +9,47 @@
 """Evaluates a trained RL policy."""
 
 import argparse
-from typing import Dict, List
+from typing import Dict, List, Union
```

Comment:
FYI in Python 3.11+, you don't actually need these typing imports

you can just do

```
Dict[str, str] = dict[str, str]
List[str] = list[str]
Union[str, float] = str | float
```